ui/src/remote.ts

```diff
@@ -44,7 +44,7 @@ declare type Starter = () => void;
 export const createStore = <T>(): Store<T> => {
   let starter: Starter | null;
   const initialState = { status: Status.NotAsked } as Data<T>
-  const internalStore = writable(initialState, () => { starter && starter() })
+  const internalStore = writable(initialState, () => { if (starter) { return starter() } })
```

Comment:
Can't you just return `starter()` here with the same effect?

Reply:
That would crash if starter is undefined.